test/worker/application_worker.js

```diff
@@ -12,7 +12,7 @@ module.exports = () => tryTests('Application Worker', tests, {
 function tests() {
 
   it('Should support loading OpenPGP.js from inside a Web Worker', async function() {
-    if (/Edge/.test(navigator.userAgent)) {
+    if (navigator && navigator.userAgent && /Edge/.test(navigator.userAgent)) {
```

Comment:
If navigator is not defined, this would still throw, so I would just do
```suggestion
    if (navigator.userAgent && /Edge/.test(navigator.userAgent)) {
```
like elsewhere, if that's sufficient.

Reply:
Good catch, fixed!